LICENSE-MIT

```diff
@@ -1,23 +0,0 @@
-Permission is hereby granted, free of charge, to any
```

Comment:
i did some snooping and this file (decapitated and weirdly small) originally came from the Rust project&hellip;

See: https://github.com/rust-lang/rust/blob/51c0db6a9118735190466d15432b5d8890bb09ee/LICENSE-MIT

i think we might need to go headless horseman on ours, too!